.github/workflows/appengine-analytics.yaml

```diff
@@ -11,7 +11,7 @@ on:
   pull_request_target:
     types: [labeled]
   schedule:
-  - cron:  '0 2 * * *'
+  - cron:  '0 0 * * 0'
```

Comment:
Moved tests from nightly to weekly until we can figure out why we're eating so many minutes.